scripts/build.sh

```diff
@@ -0,0 +1,21 @@
+#/bin/bash
+
+set -e
+
+mkdir -p ./dist/uploads
+
+# Enter the directory where Tania's Golang project is
+cd ./backend
+
+# Running any go tests
+echo "Running go test.."
+go test ./...
+
+# Build the binary
+echo "Building golang binaries..."
+go build -o ../dist/tania main.go
```

Comment:
The last `main.go` could be removed to make it less explicit and avoid future issues if something changes.